num_threads: 4
# Maximum memory to be used for buffering by the snapshotter (in mb)
# The snapshotter will take at least max_memory_mb of additional
# memory while writing a snapshot to disk!
max_memory_mb: 5000

# allowed values for bag_compression are "none", "slow", "fast"
# "none" = no compression, largest file size but very fast writing
# "slow" = best compression, smallest file size but takes a lot of time to write
# "fast" = some compression, medium file size but reasonably fast
bag_compression: "fast"
exclude_topics:
  - /vision/camera_\d+/depth/.*
  - /vision/camera_\d+/passenger_detection/debug.*
